src/MensaWidget/MWApiWrapper.class.st

```diff
@@ -0,0 +1,36 @@
+Class {
+	#name : #MWApiWrapper,
+	#superclass : #Object,
+	#category : #MensaWidget
+}
+
+{
+	#category : #openMensa,
+	#'squeak_changestamp' : 'Oscar 5/24/2024 10:51'
+}
+MWApiWrapper class >> baseEndpoint [
+ 	^ 'https://openmensa.org/api/v2'
+]
+
+{
+	#category : #openMensa,
+	#'squeak_changestamp' : 'Oscar 5/26/2024 15:47'
+}
+MWApiWrapper class >> mealsPostfix [
+	^ '/canteens/{1}/days/{2}/meals'
+]
+
+{
+	#category : #call,
+	#'squeak_changestamp' : 'Oscar 5/26/2024 15:47'
+}
+MWApiWrapper >> getMealsFrom: anOutletId on: aDate [
+	"The two attributes need to be a string"
+	
+	| url response |
+	url := self class baseEndpoint, self class mealsPostfix.
+	url := url format: {anOutletId. aDate}.
```

Comment:
you could do that in one line but not sure if that is cleaner

Reply:
I think the line will become harder to understand and more confusing. I would keep it as it is.